.changeset/grumpy-meals-kick.md

```diff
@@ -0,0 +1,6 @@
+---
+'@web/test-runner-saucelabs': minor
+'@web/test-runner-webdriver': minor
+---
+
+Upgrade WebdriverIO to v9, drop JWP capabilities
```

Comment:
"Drop"ing something feels like it should be more than a `minor` release, but I don't know Webdriver that well...

Reply:
Sure, but we are still in 0.x so technically every minor can contain breaking changes. 
I wasn't sure whether I should use `major` for this in the changeset.

Reply:
minor in 0.x is practically major, so is good

Reply:
Agreed. Thanks for clarifying.